packages/git/src/browser/style/index.css

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
@jbicker, I was unable to figure out where do we set the `id` of the left panel, so I just add this as a comment. The problem is the following: both the `Files` and the `Git` has the same `files` ID. So the browser tests are broken.

Reply:
Of course, I was wrong. I have figured out what was the problem and it is unrelated to the `id`. The ID is just fine.